anvil/distributions.py

```diff
@@ -24,40 +22,45 @@ class Gaussian:
         Mean for the distribution.
     """
 
-    def __init__(self, size_out, *, sigma=1, mean=0):
+    def __init__(self, size_out, *, mean=0, sigma=1):
+        super().__init__(mean, sigma)
```

Comment:
Do you think we should just follow PyTorch by labelling these parameters `loc` and `scale`? This would be more consistent with the way in which we wrap around `torch.optim` (i.e. not modifying the labelling of hyperparameters)